### PR TITLE
Register new package GeneticVariation.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [BioJulia/GeneticVariation.jl](https://github.com/BioJulia/GeneticVariation.jl)
Release: [v0.1.0](https://github.com/BioJulia/GeneticVariation.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/BioJulia/GeneticVariation.jl/builds/245454723)
cc: @Ward9250

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@Ward9250 This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by tkelman** [comment]
Looks like BioSymbols is imported directly, so it should be listed directly in REQUIRE instead of assuming it's present as a transitive dependency

File: GeneticVariation/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,9 @@
+julia 0.5
+Automa
+BGZFStreams
+BioCore
+BioSequences
+BufferedStreams
+Compat 0.18
```
Comment (tkelman) [inline GeneticVariation/versions/0.1.0/requires]: only need 0.17 for abstract type

**Comment by Ward9250** [comment]
@tkelman Is it imported directly? On the release-0.1 branch I removed an import of BioSymbols from src/GeneticVariation.jl because it turned out I didn't need it. It is still present on master, but when release-0.1 branch is made, any revisions on that branch will be merged back into master.

**Comment by tkelman** [comment]
> On the release-0.1 branch I removed an import of BioSymbols from src/GeneticVariation.jl because it turned out I didn't need it.

Ah yes you're right, I was looking at master, didn't realize this was tagged from a different branch

**Comment by Ward9250** [comment]
@tkelman Before this merges, I'm going hotfix BioSequences.jl and add changes to this release to address your Tuple{Int,Int} zero comment first.

**Comment by Ward9250** [comment]
@tkelman Ok I've made changes for the whole Base.zero thing. As a result this will depend on a hotfix version of BioSequences.jl, and that's reflected in REQUIRE, although I've just noticed attobot hasn't opened a PR for BioSequences.jl nor has it updated this thread.

**Comment by tkelman** [comment]
@simonbyrne did you tweak something on attobot?

**Comment by Ward9250** [comment]
I've just deleted and redone the hotfix release for BioSequences.jl and attobot still hasn't responded.

**Comment by tkelman** [comment]
Open an issue on the attobot repo?

**Comment by Ward9250** [comment]
I've done the BioSequences.jl PR myself using PkgDev.tag - PkgDev.publish in the meantime.

**Comment by Ward9250** [comment]
Ok back on track now. GeneticVariation now depends on BioSequences v0.6.2 in order to avoid the Base.zero definition, just redoing travis builds now to light that green.

**Pull request merged**